zuul.d/edpm_multinode.yaml

```diff
@@ -343,6 +343,15 @@
         dashboard_enabled: true
         cephfs_enabled: true
         ceph_nfs_enabled: true
+      # Override the Ceph container image and deploy Squid
+      cifmw_cephadm_container_ns: "quay.io/ceph"
```

Comment:
@fultonj @tosky I was thinking to switch this `multinode-hci-edpm` to `Ceph Squid`, so we have a way to do early testing upstream. 
1. Does it make sense to do this here? It would help to perform a deployment sanity check, like we are going to do in devstack (but in that case I'm not sure we're going to backport squid to Antelope) 
2. If you're positive about Squid, do we want to add here a stable tag (e.g. `v19.2`) to avoid testing the `latest` build?

I can simply remove this part here and only target something downstream. I wanted to get your opinion and feedback first because I'm potentially going to do the same thing in `install_yamls` [1] and target this new Ceph versions for tempest jobs w/ Glance, Cinder and Manila.

[1] https://github.com/openstack-k8s-operators/install_yamls/pull/952